src/frontend/src/expr/agg_call.rs

```diff
@@ -17,15 +17,56 @@ use risingwave_common::error::{ErrorCode, Result, RwError};
 use risingwave_common::types::DataType;
 use risingwave_expr::expr::AggKind;
 
-use super::{Expr, ExprImpl};
+use super::{Expr, ExprImpl, ExprRewriter};
 use crate::utils::Condition;
 
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct AggOrderByExpr {
+    pub expr: ExprImpl,
+    // `Some(true)` for ASC, `Some(false)` for DESC, `None` for default
+    pub asc: Option<bool>,
```

Comment:
How about using `Direction` or `OrderType` here?

Reply:
Now it's using `OrderType`

Reply:
I'm not sure about the field name `order_type`, maybe `direction` is better? Also not sure about `PlanAggOrderByField`, is there a better naming?